it/uk/gov/hmrc/emcstfefrontend/support/IntegrationBaseSpec.scala

```diff
@@ -24,17 +24,21 @@ import play.api.libs.json.{JsValue, Json}
 import play.api.libs.ws.{WSClient, WSRequest, WSResponse}
 import play.api.{Application, Environment, Mode}
 
+import scala.concurrent.ExecutionContext
+
 trait IntegrationBaseSpec extends SessionCookieBaker with UnitSpec with WireMockHelper with GuiceOneServerPerSuite
   with BeforeAndAfterEach with BeforeAndAfterAll {
 
+  implicit lazy val ec: ExecutionContext = app.injector.instanceOf[ExecutionContext]
+
   lazy val client: WSClient = app.injector.instanceOf[WSClient]
 
   def servicesConfig: Map[String, _] = Map(
     "microservice.services.auth.port" -> WireMockHelper.wireMockPort,
     "microservice.services.reference-data.port" -> WireMockHelper.wireMockPort,
```

Comment:
Duplicate reference data config entry, could you remove please?

Reply:
https://github.com/hmrc/emcs-tfe-frontend/pull/40/commits/db22e64b51e054f0e637067ede327ab02e2f502d